MvvmCross.Android.Support/V7.RecyclerView/MvxRecyclerAdapter.cs

```diff
@@ -167,12 +168,24 @@ public override int GetItemViewType(int position)
         protected virtual View InflateViewForHolder(ViewGroup parent, int viewType, IMvxAndroidBindingContext bindingContext)
         {
             var layoutId = ItemTemplateSelector.GetItemLayoutId(viewType);
-            return bindingContext.BindingInflate(layoutId, parent, false);
+            var view = bindingContext.BindingInflate(layoutId, parent, false);
+
+            if (layoutId == global::Android.Resource.Layout.SimpleListItem1) {
+                view.Tag = new SimpleItemTag();
+            }
+
+            return view;
+        }
+
+        public class SimpleItemTag : Object
```

Comment:
Do we really need this? Couldn't it be `global::Android.Resource.Layout.SimpleListItem1` for the tag you use for the view?

Reply:
I think it is easier to check the type in `OnBindViewHolder` then cast `Tag` it to `int`. We are never be sure what is in the `Tag`. `OnBindViewHolder` is called relatively often then I would like to do as less as possible checks.

Reply:
Technically we could also expand the `IMvxRecyclerViewHolder` interface to contain the id used for the view. Then we only need to check that and don't need to create any Java objects. We cast the holder every time anyways. I think that would be the better solution.